lib/verification.js

```diff
@@ -4,36 +4,45 @@ import { createVerify } from "node:crypto";
 
 import { request as defaultRequest } from "@octokit/request";
 
-/** @type {import('..').VerifyRequestByKeyIdInterface} */
-export async function verifyRequest(rawBody, signature, key) {
+/** @type {import('..').VerifyRequest} */
+export async function verifyRequest(rawBody, signature, keyId) {
```

Comment:
The type change is correct, but the variable name should remain `key`. This method needs the actual key, not the ID of the key